docker/Makefile

```diff
@@ -43,14 +43,22 @@ ifeq ($(IS_LATEST),true)
 	@docker tag puppet/r10k:$(version) puppet/r10k:latest
 endif
 
-test: prep
+test:
 	@bundle install --path .bundle/gems
 	@PUPPET_TEST_DOCKER_IMAGE=puppet/r10k:$(version) bundle exec rspec r10k/spec
 
-publish: prep
+publish:
 	@docker push puppet/r10k:$(version)
 ifeq ($(IS_LATEST),true)
 	@docker push puppet/r10k:latest
 endif
+	@docker pull sheogorath/readme-to-dockerhub
+	@docker run --rm \
+		-v $(PWD)/README.md:/data/README.md \
+		-e DOCKERHUB_USERNAME=$(DISTELLI_DOCKER_USERNAME) \
+		-e DOCKERHUB_PASSWORD=$(DISTELLI_DOCKER_PW) \
```

Comment:
I verified this image works on my own personal account + credentials, but haven't tested it yet with these DISTELLI_* yet.

Reply:
Also I originally had this separated into two targets -- `push-image` and `push-readme` -- and then just had:
```makefile
push-image:
    ...

push-readme:
    ...

publish: push-image push-readme
```

I kinda like that separation but could go either way.

Reply:
Hm, would probably prefer to not have these tied to the `DISTELLI_*` environment variables, can we have other vars that we pass in in our distelli-manifest.yml?

I think I like having them separated into two targets

Reply:
What's the concern with reusing these variables?

Reply:
🤷‍♀️ mostly just feels tied to the infrastructure we're using, though i guess other people won't be publishing

Reply:
Makes sense.

Ok...let's consider this as "CI-only" for now, since `publish` already is.

If we're updating the README.md more frequently than pushing an image, and we find that we want to use this make target to update DockerHub instead of copy/pasting, then we can revisit credentials here and make this target easier for local execution.